platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -475,9 +477,13 @@ MGL_EXPORT
 /**
  Offset distance of icon from its anchor.
  
+ This property is measured in points multiplied by the value of "icon-size"s.
```

Comment:
generate-style-code assumes `units` is a single plural word. The style specification should use backticks instead of double quotation marks around `icon-size`, and `propertyDoc()` should account for this case by not tacking an “s” onto the end.

Reply:
mapbox/mapbox-gl-js#5959 changes the quotation marks to backticks.